samcli/commands/remote/invoke/core/command.py

```diff
@@ -0,0 +1,148 @@
+                        ),
+                    ]
+                )
+            with formatter.indented_section(
+                name="Invoke lambda function using lambda ARN and get full boto response", extra_indents=1
+            ):
+                formatter.write_rd(
+                    [
+                        RowDefinition(
+                            text="\n",
+                        ),
+                        RowDefinition(
+                            name=style(
+                                f"${ctx.command_path} arn:aws:lambda:us-west-2:123456789012:function:my-function -e <>"
+                                f" --output-format raw"
+                            ),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                    ]
+                )
+            with formatter.indented_section(
+                name="Asynchronously invoke lambda function with additional boto parameters", extra_indents=1
+            ):
+                formatter.write_rd(
+                    [
+                        RowDefinition(
+                            text="\n",
+                        ),
+                        RowDefinition(
+                            name=style(
+                                f"${ctx.command_path} HelloWorldFunction -e <> "
+                                f"--parameter InvocationType=Event --parameter Qualifier=MyQualifier"
+                            ),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                    ]
+                )
+            with formatter.indented_section(
+                name="Dry invoke a lambda function to validate parameter values and user/role permissions",
+                extra_indents=1,
+            ):
+                formatter.write_rd(
+                    [
+                        RowDefinition(
+                            text="\n",
+                        ),
+                        RowDefinition(
+                            name=style(
+                                f"${ctx.command_path} HelloWorldFunction -e <> " f"--parameter InvocationType=DryRun"
+                            ),
+                            extra_row_modifiers=[ShowcaseRowModifier()],
+                        ),
+                    ]
+                )
+
+    @staticmethod
+    def format_acronyms(formatter: RemoteInvokeCommandHelpTextFormatter):
+        with formatter.indented_section(name="Acronyms", extra_indents=1):
+            formatter.write_rd(
+                [
+                    RowDefinition(
+                        name="ARN",
+                        text="Amazon Resource Name",
+                        extra_row_modifiers=[ShowcaseRowModifier()],
+                    ),
+                    RowDefinition(
+                        name="CFN",
+                        text="CloudFormation",
```

Comment:
AWS Cloudformation, do we reference it anywhere else?

Reply:
We reference it in the help text for the command: `Invoke or send an event to cloud resources in your CFN stack`

Reply:
Actually, I will remove it as the command is not restricted to CFN stack.